src/vs/base/browser/cssValue.ts

```diff
@@ -2,9 +2,16 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
+import { Color } from '../common/color.js';
 import { FileAccess } from '../common/network.js';
 import { URI } from '../common/uri.js';
 
+export type CssFragment = string & { readonly __cssFragment: unique symbol };
+
+function asFragment(raw: string): CssFragment {
+	return raw as CssFragment;
+}
```

Comment:
Is the purpose of this to ensure that certain APIs only take a `CssFragment` not a string?

Reply:
Yes, eventually I'd like to create helpers that create a new `<style>` element that only takes a `CssFragment `. That way any css string in our js will have had to go through these functions